src/common/storage/storageclasses/editorstorage.js

```diff
@@ -464,6 +464,17 @@ define([
             }
         };
 
+        this.squashCommits = function (projectId, fromCommit, toCommitOrBranch, msg, callback) {
```

Comment:
Would be nice to have this on project/userproject too.
Additionally on the REST api.